app/models/admin_user.rb

```diff
@@ -0,0 +1,7 @@
+class AdminUser < ActiveRecord::Base
```

Comment:
Yep, I prefer the Super Admin to be completely separated from the normal User. I don't want to add a flag, roles or anything. A Team Administrator is a User with an Enrollment to a Team and a is_admin flag there. But a Super Admin has no ties to any Team and he can only log in to Active Admin.